html/pages/device.inc.php

```diff
@@ -435,6 +435,14 @@
                 </a>
                 </li>';
         }
+        
+        if ($config['enable_ports_nac'] == 1) {
```

Comment:
Convert this check to a dbFetch to see if data exists in the new tables for this device.

Reply:
I have created a new module called Cisco-NAC
<img width="1422" alt="screen shot 2018-11-06 at 23 48 51" src="https://user-images.githubusercontent.com/15846428/48105422-9dd9e800-e21e-11e8-99da-103f9ca8c022.png">